content/docs/reference/cli-create-app.md

```diff
@@ -12,7 +12,17 @@ updatedOn: '2024-07-12T11:16:39.827Z'
 
 ## The `create-app` command
 
-This command initializes a new Neon project.
+The `create-app` command initializes a new Neon project and bootstraps a full-stack application using your preferred package manager. The command supports the following stack components:
+
+- **Package Manager**: Choose from `npm`, `pnpm`, and `bun`
+- **Frameworks**: Choose from `Next.js`, `SvelteKit`, and `Nuxt.js`
+- **ORM**: Choose from `Drizzle` and `Prisma`
```

Comment:
Only Drizzle right now; Prisma gets 'coming soon' when you arrow down to it

Reply:
fixed.